whois-api/src/test/java/net/ripe/db/whois/api/AbstractIntegrationTest.java

```diff
@@ -32,4 +34,23 @@ public void stopServer() throws Exception {
     public int getPort() {
         return jettyBootstrap.getPort();
     }
+
+   /**
+    * This method can be called anywhere in a derived test class to
+    * be able to debug the server on a local development machine.
+    */
+    protected synchronized void stopExecutionHereButKeepTheServerRunning() {
+        Instant start = Instant.now();
+
+        while (!Thread.currentThread().isInterrupted()) {
+            try {
+                Instant end = Instant.now();
+                Duration timeElapsed = Duration.between(start, end);
+                System.out.println(String.format("Server listening for %d minutes on port %d", timeElapsed.toMinutes(), getPort()));
+                wait(60000);
+            } catch (InterruptedException ex) {
+                Thread.currentThread().interrupt();
```

Comment:
is the interrupt() needed if the wait threw an InterruptedException? The thread isInterrupted() should still be true, even without the interrupt() ?